app/recordtransfer/templates/recordtransfer/signupform.html

```diff
@@ -1,28 +1,22 @@
-{% extends 'recordtransfer/base.html' %}
+{% extends "recordtransfer/base.html" %}
 {% load static %}
 {% load i18n %}
-
-{% block title %}{% trans "Create Account" %}{% endblock %}
-
+{% block title %}
+    {% trans "Create Account" %}
+{% endblock title %}
 {% block content %}
     <div class="account-form-container">
-        <div class="account-page-title title-text">
-            {% trans "Create a New Account" %}
-        </div>
-
+        <div class="account-page-title title-text">{% trans "Create a New Account" %}</div>
         {% if form.non_field_errors %}
-            <div class="account-error">
-                {% trans "The form was not valid." %}
-            </div>
+            <div class="account-error">{% trans "The form was not valid." %}</div>
         {% endif %}
-
         <form method="post" action="{% url 'recordtransfer:create_account' %}">
             {% csrf_token %}
             {% for field in form %}
                 {% if field.errors %}
                     {% for error in field.errors %}
                         <div class="flex-error">
-                            <label></label>
+                            {{ field.label_tag }}
```

Comment:
This needs to be an empty label, because the way it's set right now, you get an extra label in the template which doesn't look right when there's an error:

![Screenshot 2025-05-13 081849](https://github.com/user-attachments/assets/9427f097-f74d-4d43-b957-4bbe31c8a975)

```suggestion
                            <label></label>
```

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/561/commits/fb2aa83c8e3a11340cbdc5b35878dbd226b64431